File: Patika.dev Coderbyte Assessments (Weekly)/Week 2/Python/StepWalking.py
def StepWalking(num):
    # code goes here
    if num == 1:
        return 1
    elif num == 2:
        return 2
    else:
        return StepWalking(num - 1) + StepWalking(num - 2)


# keep this function call here 
print(StepWalking(input()))

File: Patika.dev Coderbyte Assessments (Weekly)/Week 2/Java/StepWalking.java
import java.util.*;
import java.io.*;

class Main {

  public static int StepWalking(int num) {
    // code goes here
    if (num == 1) {
      return 1;
    }
    else if (num == 2) {
      return 2;
    }
    else {
      return (StepWalking(num - 1) + StepWalking(num - 2));
    }
  }

  public static void main (String[] args) {
    // keep this function call here
    Scanner s = new Scanner(System.in);
    System.out.print(StepWalking(s.nextLine()));
  }

}